pipeline/src/node_properties/csi_camera_properties.rs

```diff
@@ -15,6 +15,13 @@ pub struct CsiCameraProperties {
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct CsiCameraRuntime {
+    /// Camera URI.
+    ///
+    /// This field is set by API server via camera lookup by `source` field.
```

Comment:
the part after "via.." is internal detail of API server and don't think it needs to be mentioned here. Same for other similar doc comments.